resources/provider/migrations/postgres/24_v0.10.16.up.sql

```diff
@@ -1,6 +1,58 @@
 -- Autogenerated by migration tool on 2022-04-04 17:12:37
 -- CHANGEME: Verify or edit this file before proceeding
 
+-- Resource: codepipeline.pipelines
+CREATE TABLE IF NOT EXISTS "aws_codepipeline_pipelines" (
+                                                            "cq_id" uuid NOT NULL,
+                                                            "cq_meta" jsonb,
+                                                            "account_id" text,
+                                                            "region" text,
+                                                            "tags" jsonb,
+                                                            "created" timestamp without time zone,
+                                                            "arn" text,
+                                                            "updated" timestamp without time zone,
+                                                            "name" text,
+                                                            "role_arn" text,
+                                                            "artifact_store_location" text,
+                                                            "artifact_store_type" text,
+                                                            "artifact_store_encryption_key_id" text,
+                                                            "artifact_store_encryption_key_type" text,
+                                                            "artifact_stores" jsonb,
+                                                            "version" integer,
+                                                            CONSTRAINT aws_codepipeline_pipelines_pk PRIMARY KEY(arn),
+                                                            UNIQUE(cq_id)
+);
+CREATE TABLE IF NOT EXISTS "aws_codepipeline_pipeline_stages" (
+                                                                  "cq_id" uuid NOT NULL,
```

Comment:
I think you're missing the stage ordering information here?

Reply:
I see there's `run_order` in stage_actions, but don't the stages themselves have order within a pipeline? I don't know, just asking. Terraform `stage` blocks also don't seem to have explicit ordering apart from being separate blocks.

Reply:
Great call out. I think you are right that the stage ordering is implicit to how the blocks are ordered. That is also how it is in Cloudformation. Looking at the [CLI](https://docs.aws.amazon.com/cli/latest/reference/codepipeline/update-pipeline.html) there is no way to update a single stage, you have to specify all of the stages. As such we should create a column to specify the index/order of each stage